README.md

```diff
@@ -324,6 +333,19 @@ Types are exported from `./lib/types/[node/web]/index.d.ts` and should be automa
   });
   ```
 
+  - `topUpWithTokens({ tokenAmount, feeMultiplier })` - Tops up the connected wallet with Credits by submitting a payment transaction for the token amount to the Turbo wallet and then submitting that transaction id to Turbo Payment Service for processing.
+    - The `tokenAmount` is the amount of tokens in the token type's smallest unit value (e.g: Winston for arweave token type) to fund the wallet with.
+    - The `feeMultiplier` is the multiplier to apply to the reward for the transaction to modify its chances of being mined. Credits will be added to the wallet balance after the transaction is confirmed on the given blockchain.
```

Comment:
May want to indicate that the default is 1.0.

Reply:
And that it's optional

Reply:
c31d1d6